Source/DafnyLanguageServer/Workspace/DocumentManagerDatabase.cs

```diff
@@ -71,18 +72,18 @@ public class DocumentManagerDatabase : IDocumentDatabase {
       return false;
     }
 
-    public Task<DafnyDocument?> GetResolvedDocumentAsync(TextDocumentIdentifier documentId) {
+    public Task<CompilationView?> GetResolvedDocumentAsync(TextDocumentIdentifier documentId) {
```

Comment:
I feel weird that two similar named methods `GetResolvedDocumentAsync` and `GetLastDocumentAsync` now return two completely different classes. Maybe you could rename these methods so that it's obvious they return things that are different?